test/integration/dialog.test.js

```diff
@@ -27,60 +27,102 @@ describe('dialogs', () => {
   describe('alert', () => {
     beforeEach(() => browser.clickOn('.link_to_open_an_alert'));
 
-    it('can get an alert text', async () => {
-      const text = await browser.getAlertText();
-      await browser.acceptAlert();
-      assert.equal('Alert text was not found', 'An alert!', text);
+    it('can get an alert text', done => {
```

Comment:
Why'd these all change back to testing callback mode?

Reply:
I discovered that `acceptAlert()` and `dismissAlert()` are not part of the promise chain, but use WD commands directly.
Locally those tests run fine with `aync/await` but they are somewhat flaky in Travis

I'll promisify the them in a follow up.